dspace/modules/xmlui-mirage2/src/main/webapp/themes/vtmirage2/styles/_style.scss

```diff
@@ -62,7 +62,9 @@ white-space:pre-line;
 word-break:break-all;
 width:200px;
 }
-
+.filename-word-wrap {
```

Comment:
I meant to comment on this specific file, where some styles are indented below the selectors, and some are not.